### DISCUSSION TITLE
Change the name of the package

### DISCUSSION THREAD

**Discussion by hgalytoby** [python-poetry/poetry#7951] [Q&A] [answered]
Poetry version: 1.4.1

When I install with the following it goes fine, but I want to change the name now.

`packages = [{ include = "AggregateRoot" }]`

can be used like this.

```py
from AggregateRoot import *
```

Now I want to change the name of the package

For example use like this

```py
from MyTool import *
```

I can't change the name of the folder, so how do I change `pyproject.toml.`?

**Comment by finswimmer** [comment]
Hey,

if you want to change the name, that is used in the `import` statement, you have to change the name of the folder. This is because the `import` statement is looking for a folder with this name and not for the official name of the package itself.

fin swimmer

**Reply by hgalytoby** [reply, marked as answer]
Thank you for your reply, I had to modify the file name.
